### DISCUSSION TITLE
Incorrect amount decimal separator and date format

### DISCUSSION THREAD

**Discussion by michelebossa** [firefly-iii/firefly-iii#5564] [Firefly III] [answered]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [ ] ... it's [definitely me, not you](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I have an difference between the showing of data and the insert pages about the local currency and date format handling.
Example of Amount with coma on decimal part 
<img width="178" alt="image" src="https://user-images.githubusercontent.com/38975058/149751884-4a2135fd-a36d-4176-aa6a-fb42a5980e14.png">

Douring the creation of new transcation will have to put a dot and not a comma in decimal part (it is not correct in euro handling i supose)
<img width="193" alt="image" src="https://user-images.githubusercontent.com/38975058/149751703-15ac0c9d-a12b-4c16-bd37-9708727d0397.png">

Data is month/day/year but in italy we use day/month/years
<img width="258" alt="image" src="https://user-images.githubusercontent.com/38975058/149751922-cbdf99cd-65dc-4700-95b0-4b277a128378.png">

I have setted 
DEFAULT_LANGUAGE=it_IT
DEFAULT_LOCALE=equal

### Expected behaviour

As data showed i'd like to have in creation section the correct local handling of amount and data.

### Debug information

N/A

**Comment by JC5** [comment, marked as answer]
These are set by your browser, I have no influence on them.

**Reply by michelebossa** [reply]
And there is no action we can made to fix this?
Usually my browser work on other international site

**Reply by JC5** [reply]
If you set your browser to your preferred locale, the inputs should update.

**Reply by michelebossa** [reply]
I have changed italien as Display languages under chrome setting and will FIX thanks a lot
